### PR TITLE
Recolored clothes properly show up on avatar

### PR DESCRIPTION
**Pull request by Crystalwarrior**
Title. Makes recolored clothes actually show up on person correctly.
Also removed huge amounts of hardcode from washing machine and made it simply recolor clothes, tell me if I should revert that one.

### CODE REVIEW

File: code/game/objects/items/weapons/cigs_lighters.dm [hunk 1]
```diff
@@ -116,6 +116,7 @@ CIGARETTE PACKETS ARE IN FANCY.DM
 	var/chem_volume = 30
 	heat = 1000
 	burn_state = 0
+	can_be_washed = 0
```
Comment (Carbonhell) [inline code/game/objects/items/weapons/cigs_lighters.dm:119]: remove

**Comment by danhibiki** [comment]
This good to go?

**Comment by Crystalwarrior** [comment]
Yeppers

**Comment by Nebyab** [comment]
Yeah sure

**Pull request merged**